ci/k6-ingestion-test.js

```diff
@@ -1,6 +1,17 @@
 import http from 'k6/http'
-import {check, group, sleep} from 'k6'
+import {check, sleep, fail} from 'k6'
 import { Gauge } from 'k6/metrics'
+import { URL } from 'https://jslib.k6.io/url/1.0.0/index.js';
```

Comment:
This import is weird. Context?

Reply:
I can move it back to string if you prefer. I was following the [docs](https://k6.io/docs/examples/urls-with-query-parameters/)

Reply:
Nit: Last remaining semicolon :D

Reply:
🤦